src/errors.rs

```diff
@@ -13,6 +13,12 @@ pub enum OrandaError {
     #[error(transparent)]
     StripPrefixError(#[from] std::path::StripPrefixError),
 
+    #[error(transparent)]
```

Comment:
i think when we add errors these days we should try to add full errors w/messages IMO and avoid purely transparent ones unless the error is uniquely good

Reply:
yeah, it's better, I removed this and if there is an error parsing the date I also do not think it makes sense to break everything, I'll just show the original date we have